### DISCUSSION TITLE
A Strange Error pt.2

### DISCUSSION THREAD

**Discussion by onurjkk** [msiemens/tinydb#427] [Q&A]
in the console it gives me the error that "inventory" does not have the "search" command but as you can see from the images "inventory" is a table of the db and also when I try to do "insert, search, update" it gives me the same error. Can you help me?
![error](https://user-images.githubusercontent.com/76625957/132131030-a8e7c9a7-80ba-43f8-8333-da51d5e3749d.jpg)
![code1](https://user-images.githubusercontent.com/76625957/132131032-0582094a-3a99-40af-8610-9997682d9538.png)
![code2](https://user-images.githubusercontent.com/76625957/132131036-0ad6d6d5-264d-4d1c-88c2-b0ad7439c0d8.png)

**Comment by eugene-eeo** [comment]
Are you sure `inventory` isn't being reassigned somewhere?

**Reply by onurjkk** [reply]
yes

**Reply by msiemens** [reply]
That's the only explanation I can see, too. At some point, `inventory` (which should be a `tinydb.table.Table` or `tinydb.database.Database`) object) becomes a Discord `Command` object.

@onurtvb Maybe you can put `print()` statements in your code to determine at which point `inventory` switches from being a TinyDB object to being a Discord object. Apart from that, I don't think there's much we can do without seeing the full source code.
